src/sentry/static/sentry/app/components/charts/simpleTableChart.tsx

```diff
@@ -68,10 +70,10 @@ const StyledPanelTable = styled(PanelTable)`
   border-right: 0;
   border-bottom: 0;
 
-  /* align height with charts */
-  height: 190px;
-  overflow: scroll;
   margin: ${space(1)} 0 0;
+  ${/* sc-selector */ PanelTableHeader} {
```

Comment:
Is this intentional? Is this some sort of macro?

Reply:
It is a [hint to stylelint/eslint to not yell about selectors](https://styled-components.com/docs/tooling#interpolation-tagging) with objects in them.